### DISCUSSION TITLE
Model finetuning on continuous data

### DISCUSSION THREAD

**Discussion by mahadev0811** [facebookresearch/detectron2#5164] [General]
Basically i took a pretrained faster rcnn model and trained it on custom dataset (4 classes ) with around 20000 images and saved the best checkpoint as "obj_finetune-1.pth" , now a new dataset of around 5000 images (but only 2 classes) has been generated and I want to update the previously finetuned model  "obj_finetune-1.pth" with new dataset as well
How can I do this and make sure model's accuracy to detect all 4 classes do not drop upon further finetunings
